mslice/plotting/plot_window/slice_plot.py

```diff
@@ -216,7 +220,7 @@ def toggle_overplot_line(self, key, recoil, checked, cif_file=None):
         self._canvas.draw()
 
         # Since some methods in the plot figure manager do not check if the plot handler is a slice plot or cut plot
```

Comment:
I'm not sure this comment fully explains what is happening here. As far as I can tell we call `self.manager.report_as_current()` at the start of the function and this just resets the state - is that correct? If so can we update the comment to reflect this?

Reply:
My understanding is that `self.manager.report_as_current()` tells the global figure manager that the plot figure contained in `self.manager` is where "plot instructions" should be directed. Is this right?
I'll change the comment to `# Reset current active figure`?